server.js

```diff
@@ -1,21 +1,55 @@
 const express = require('express');
 const expressHandlebars = require('express-handlebars');
 const vhost = require('vhost');
-const setBananaI18n = require('./middleware/setBananaI18n.js');
 const rootRoutes = require('./rootRoutes');
 require('dotenv').config();
 require('handlebars-helpers')();
 
+const handlebars = expressHandlebars.create({
```

Comment:
I probably don't have any more experience with using Handlebars and Express this way than you do, but what about moving these helpers to `/viewHelpers` (like our other helpers)? The difference, of course, is that you're applying these to the engine, not to each route, but what you could do is define all three in one `viewHelpers/navHelpers.js`, then

    import { createCountryDropdownHref, createLocaleDropdownHref, createNavbarItemHref } from './viewHelpers/navHelpers.js'

then

    const handlebars = expressHandlebars.create({
      helpers: {
        createCountryDropdownHref,
        createLocaleDropdownHref,
        createNavbarItemHref
      }
    })

feel free to ignore, but I *think* these functions are the architectural equivalent of hte other ones we're storing in `/viewHelpers`, so they should probably be in the same place?